ReplicationAcceptance/ReplicationAcceptance/CloudantReplicationBase.m

```diff
@@ -12,15 +12,20 @@
 #import "CloudantSyncEncryption.h"
 
 #import "ReplicationSettings.h"
+#import "CDTLogging.h"
 
 #import <UNIRest.h>
 
+
+
 @implementation CloudantReplicationBase
 
+
 - (void)setUp
 {
     [super setUp];
     // Put setup code here; it will be run once, before the first test case.
+    CDTChangeLogLevel(CDTREPLICATION_LOG_CONTEXT, DDLogLevelAll);
```

Comment:
Removed in 8ae06ed. @tomblench I think we should file a ticket to add it back in with the code to control it based on the environment variable.